_docs/_latest/configure/config-validator/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Config Validator
+order: 700
+---
+
+# {{ page.title }}
+
+This page lists the steps to set up Config Validator Scanner. 
+
+---
+
+## **Setting up Config Validator Scanner**
+
+In your `main.tf` file, set the `config_validator_enabled` variable in the 
+Forseti Terraform module to `"true"`:
+
+```
+module "forseti" {
+  source                   = "terraform-google-modules/forseti/google"
+  project_id               = "PROJECT_ID"
+  org_id                   = "ORG_ID"
+  domain                   = "DOMAIN"
+  
+  ...
+  
+  config_validator_enabled   = "true"
+  config_validator_image     = "CONFIG_VALIDATOR_IMAGE"
+  config_validator_image_tag = "CONFIG_VALIDATOR_IMAGE_TAG"
+}
+```
+
+**Note:**
+- You will be notified when Config Validator violations are found as 
+`config_validator_violations_should_notify` is set to `”true”` by default.
+- `config_validator_image` and `config_validator_image_tag` need to be set 
```

Comment:
I think this is mostly ok, but people need to be aware of the consequences of updating CV image to something that is possibly not supported. If there are any proto updates required by CV, then we need to determine how to handle that. This issue is to determine/document how to handle this: https://github.com/forseti-security/forseti-security/issues/3140.

Reply:
Good point, added a note to check in with the Forseti team if they want to use a specific CV image/tag.